src/test/resources/application.conf

```diff
@@ -89,25 +88,73 @@ cnp_sanbox {
   createCaseUrl = "caseworkers/6687/jurisdictions/PROBATE/case-types/GrantOfRepresentation/cases"
 }
 
-cnp_aat {
-  jurisdiction = "sscs" //TODO get the jurisdiction from the url
+cnp_sprod {
+  jurisdiction = "autotest1"
```

Comment:
I'm quite sure this parameter is no longer needed. Can you please remove it and verify that ev still work?

Reply:
Removed Jurisdiction parameter and tested OK locally in sprod environment